packages/http/src/mocker/generator/HttpParamGenerator.ts

```diff
@@ -0,0 +1,64 @@
+import { IHttpOperation, IHttpParam, IHttpPathParam } from '@stoplight/types';
+import { INodeExample, INodeExternalExample } from '@stoplight/types';
+import * as Array from 'fp-ts/lib/Array';
+import * as Option from 'fp-ts/lib/Option';
+import { pipe } from 'fp-ts/lib/pipeable';
+import { JSONSchema } from '../../types';
+import { generate as generateDynamicExample } from './JSONSchema';
+
+function improveSchema(schema: JSONSchema) {
+  const newSchema = { ...schema };
+
+  if (newSchema.type === 'integer') {
```

Comment:
This part of code is valid for integers as well

```suggestion
  if (newSchema.type === 'integer' || newSchema.type === 'number') {
```

Reply:
👍